lib/index.js

```diff
@@ -158,27 +161,40 @@ var productLabelMap = {
 Quantcast.prototype.orderCompleted = function(track) {
   var labels = this._labels(track);
 
-  var category = safe(track.category());
-  if (this.options.advertise && category) {
-    labels += ',_fp.pcat.' + category;
-  }
-
-  var repeat = track.proxy('properties.repeat');
-  if (this.options.advertise && typeof repeat === 'boolean') {
-    labels += ',_fp.customer.' + (repeat ? 'repeat' : 'new');
+  if (this.options.advertise) {
+    var category = safe(track.category());
+    if (category) labels += ',_fp.pcat.' + category;
+
+    var repeat = track.proxy('properties.repeat');
+    if (typeof repeat === 'boolean') labels += ',_fp.customer.' + (repeat ? 'repeat' : 'new');
+
+    if (this.options.advertiseProducts) {
+      var products = track.products();
+
+      products.forEach(function(product) {
+        // only include products with an ID (as required by the spec)
+        if (!objCase.find(product, 'product_id')) return;
+
+        Object.keys(productLabelMap).forEach(function(key) {
+          var value = objCase.find(product, key);
+          var label = productLabelMap[key];
+          if (value) labels += ',_fp.pcat.' + label + '.' + value;
```

Comment:
is changing from `=` to `.` not a breaking change?

Reply:
This change was requested by the Quantcast team after this was originally shipped last week, and they seem to have mis-communicated the `=` when they requested it previously. I'm treating this as a bugfix as a result.